src/features/iam/tree/iamExplorerProvider.ts

```diff
@@ -38,6 +40,11 @@ export class IamExplorerProvider implements ExplorerProvider {
 
   private async getProjects() {
     const projects = await iamState.listProjects()
+    const currentUserId = configVaultService.getCurrentUserID()
+    const activeProjectId = currentUserId
+      ? configVaultService.getActiveProjectId(currentUserId)
+      : null
+    setActiveProject(activeProjectId ?? projects[0].projectId)
```

Comment:
it may fail if users have no projects. Add `?` pls
```
projects[0]?.projectId
```

Reply:
hm there could be users without project right now, so need to check it anyways